File: ExportToExcel/Models/DetectOperation.vb
Namespace Models
    Public Class DetectOperation

        Private _id As Integer
        Private _baseImage As String
        Private _detectedImage As String
        Private _operationTime As Long

        Public Property Id As Integer
            Get
                Return _id
            End Get
            Set(value As Integer)
                _id = value
            End Set
        End Property

        Public Property BaseImage As String
            Get
                Return _baseImage
            End Get
            Set(value As String)
                _baseImage = value
            End Set
        End Property

        Public Property DetectedImage As String
            Get
                Return _detectedImage
            End Get
            Set(value As String)
                _detectedImage = value
            End Set
        End Property

        Public Property OperationTime As Long
            Get
                Return _operationTime
            End Get
            Set(value As Long)
                _operationTime = value
            End Set
        End Property
    End Class
End Namespace

File: FaceDetectService/DAL/Models/detectoperation.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace FaceDetectService.Models
{
    [Table("detectoperation")]
    public class detectoperation
    {
        private int _id;
        private string _baseImage;
        private string _detectedImage;
        private long _operationTime;

        public int id { get => _id; set => _id = value; }
        public string baseImage { get => _baseImage; set => _baseImage = value; }
        public string detectedImage { get => _detectedImage; set => _detectedImage = value; }
        public long operationTime { get => _operationTime; set => _operationTime = value; }
    }
}
